CHANGELOG.md

```diff
@@ -49,6 +49,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur#2827](https://github.com/cyberark/conjur/pull/2827)
 
 ### Security
+- Upgrade google/cloud-sdk in ci/test_suites/authenticators_k8s/dev/Dockerfile/test
+  to use latest version (448.0.0)
+  [cyberark/conjur#2972](https://github.com/cyberark/conjur/pull/2972)
```

Comment:
Usually the changelog is reserved for changes that have an end-user impact. Giving this is a container for tests, it's not actually needed here.

Reply:
Fair - are you suggesting I remove it, or just commenting for next time? (I included it largely because the CVE is getting publicity, but since it's only in test files, it probably doesn't matter either way.)

Reply:
No, I don't think you need another push just to remove it. If you end up making any other updates you could pull it out, but if this build is good, it's fine to stay. Thanks!